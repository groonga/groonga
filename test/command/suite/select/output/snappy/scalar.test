table_create Entries TABLE_PAT_KEY ShortText
column_create Entries content COLUMN_SCALAR|COMPRESS_SNAPPY Text

load --table Entries
[
  {
    "_key": "Groonga",
    "content": "I found Groonga that is a fast fulltext search engine!"
  },
  {
    "_key": "Mroonga",
    "content": "I found Mroonga that is a MySQL storage engine to use Groonga!"
  }
]

select Entries
