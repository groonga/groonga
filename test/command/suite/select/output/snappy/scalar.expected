table_create Entries TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Entries content COLUMN_SCALAR|COMPRESS_SNAPPY Text
[[0,0.0,0.0],true]
load --table Entries
[
  {
    "_key": "Groonga",
    "content": "I found Groonga that is a fast fulltext search engine!"
  },
  {
    "_key": "Mroonga",
    "content": "I found Mroonga that is a MySQL storage engine to use Groonga!"
  }
]
[[0,0.0,0.0],2]
select Entries
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        2
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "ShortText"
        ],
        [
          "content",
          "Text"
        ]
      ],
      [
        1,
        "Groonga",
        "I found Groonga that is a fast fulltext search engine!"
      ],
      [
        2,
        "Mroonga",
        "I found Mroonga that is a MySQL storage engine to use Groonga!"
      ]
    ]
  ]
]
